video/probe.go

```diff
@@ -231,3 +241,22 @@ func parseFps(framerate string) (float64, error) {
 
 	return float64(num) / float64(den), nil
 }
+
+func findFormat(format string) string {
+	actualFormats := strings.Split(format, ",")
+	for _, f := range supportedFormats {
+		if containsStr(actualFormats, f) {
+			return f
+		}
+	}
+	return actualFormats[0]
```

Comment:
What if `actualFormats` is empty? Or it won't ever happen?

Reply:
Good question, I just checked and we should be ok as strings.Split will return `[""]` if `format` is empty, which will give us the same behaviour as before setting the format to `""`.